src/scripts/oauth_tables_script.sql

```diff
@@ -0,0 +1,110 @@
+
+DROP TABLE IF EXISTS app_clients CASCADE;
+
+CREATE TABLE app_clients(
+             id BIGSERIAL NOT NULL,
+             client_name VARCHAR(50) UNIQUE NOT NULL,
+             client_id VARCHAR(50) UNIQUE NOT NULL,
```

Comment:
Client id can be bigger than that

Reply:
If you are using status this cannot be unique

Reply:
done